.github/actions/get-ado-token/action.yml

```diff
@@ -1,34 +0,0 @@
-name: Get Azure DevOps Access Token

```

Comment:
This file ends up no longer being needed because the pipeline itself has access to the necessary ADO feeds.